v21.2/create-and-configure-changefeeds.md

```diff
@@ -7,20 +7,23 @@ docs_area: stream_data
 
 Core and {{ site.data.products.enterprise }} changefeeds offer different levels of configurability. {{ site.data.products.enterprise }} changefeeds allow for active changefeed jobs to be [paused](#pause), [resumed](#resume), and [canceled](#cancel).
 
-## Create a changefeed (Core)
+When creating a changefeed, it's important to consider the number of changefeeds versus the number of tables to include in a single changefeed:
 
-A core changefeed streams row-level changes to the client indefinitely until the underlying connection is closed or the changefeed is canceled.
+  - Changefeeds do not share internal buffers, so each running changefeed will increase total memory usage. To watch multiple tables, we recommend creating a changefeed with a comma-separated list of tables.
```

Comment:
2¢, I think “internal buffers” is more detail than we need, it’s sufficient to say that each changefeed has its own memory overhead.

On which, @HonoreDB do you know the per-changefeed memory overhead? Is it worth mentioning?

Reply:
Yes, definitely a good point. That first portion was existing content, but yes, it is a little more "internal-ly" than we'd normally get in docs. I like your suggestion here 👍